### PR TITLE
W-12472939-monitoring-applications-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by failup — DISMISSED** [review]

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -26,13 +26,14 @@
...
- ** xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
- ** xref:use-log4j-appender.adoc[Use a Log4j Appender]
  ** xref:protect-app-properties.adoc[Protect App Property Values]
  ** xref:manage-secure-properties.adoc[Protect App Property Values with rtfctl]
  ** xref:app-versioning.adoc[Review Application Versioning]
  ** xref:app-to-app-requests.adoc[Send Application-to-Application Requests]
  ** xref:proxy-deploy-runtime-fabric.adoc[Deploy API Proxies]
+* xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
+ ** xref:use-anypoint-monitoring.adoc[Use Anypoint Monitoring]
+ ** xref:use-log4j-appender.adoc[Use a Log4j Appender]
```
Comment (SparshAgarwal) [inline modules/ROOT/nav.adoc:36]: Change title to "Export telemetry externally"

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 2]
```diff
@@ -1,103 +1,12 @@
...
-== Anypoint Visualizer
-
-xref:visualizer::index.adoc[Anypoint Visualizer] displays views of different aspects of an application network graph. 
-You can use the graph to explore your application network, identify problems, and make decisions.
-
-[WARNING]
-====
-By default, Anypoint Visualizer is disabled when deploying an application to Mule runtime 4.x. 
-See xref:visualizer::setup.adoc#enable-header-injection[Enable Header Injection] for more information.
+== Log4j Appender
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/manage-monitor-applications.adoc:8]: Change title to "Export telemetry externally"

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 3]
```diff
@@ -1,103 +1,12 @@
...
+You can use a Log4j appender to integrate your logging system with applications deployed to Runtime Fabric. For information about configuring log4j2.xml, see the Log4j Configuration SyntaxLeaving the Site Refer to xref:use-log4j-appender.adoc[]
 
-Due to the impact on performance, enabling Anypoint Visualizer for application deployments with less than 0.2 vCPU is 
-not supported.
-====
+== Otel Tracing
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/manage-monitor-applications.adoc:11]: Move this section to another PR, this feature is not going out now

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 4]
```diff
@@ -0,0 +1,103 @@
+= Use Anypoint Monitoring
+
+Applications and API gateways deployed to Runtime Fabric include native support for Anypoint Monitoring to allow viewing metrics within Anypoint Platform.
+
+[NOTE]
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:5]: remove this note

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+====
+If you are using a version of Anypoint Runtime Fabric older than 1.8.50, use port 5044 instead of port 443. Use the following domains:
+
+* US control plane: `dias-ingestor-nginx.prod.cloudhub.io` 
+* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
+====
+
+== Disable Anypoint Monitoring for an Application
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:31]: Move this section to end of page
Add “(Not recommended)” at beginning of title
Add “note: For compliance or geospecific requirement if your team/organization cannot send metrics to Anypoint control plane. You can disable Anypoint metrics if you are not contractually obligated to send these metrics for Anypoint usage and billing purposes. “

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+When you deploy an application to Runtime Fabric, Anypoint Monitoring is enabled by default. To disable Anypoint 
+Monitoring, pass the following custom property when deploying your application:
+
+[source,copy]
+----
+anypoint.platform.config.analytics.agent.enabled=false
+----
+
+[NOTE]
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:41]: Move this note to logs section

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+In some cases, logs with multiple lines might not be fully transmitted to Anypoint Monitoring. This can occur if 
+multiple lines of log output are split across two files due to log rotation inside Runtime Fabric.
+
+== Metrics
+
+Each Mule application and API gateway replica runs a metrics agent that collects and transmits metrics data to 
+Anypoint Monitoring via secure mutual TLS encryption. Each replica can cache up to 100 megabytes of metrics, after which the metrics data is rotated to provide storage for more recent data. In the event that metrics cannot be sent to Anypoint Monitoring, this typically provides at least 4 days worth of storage per replica. Note that custom metrics might consume more storage.
+
+[NOTE]
+--
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:51]: Change heading to “Anypoint runtime fabric updates(listed below) do not impact or restore monitoring metrics of applications deployed on runtime fabric”

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+[NOTE]
+====
+The log forwarding agent can transmit up to 300 kilobytes of log data per second. You should not deploy more than
+50 replicas per worker node.
+====
+
+You can enable log forwarding and set log levels when you xref:deploy-to-runtime-fabric.adoc#configure-logging-and-log-levels[deploy applications in Runtime Manager]. 
+
+==== Anypoint Monitoring Log Forwarding Prerequisites
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:78]: Change title to "Resource and permission requirements for Anypoint Monitoring agent"

File: modules/ROOT/pages/use-anypoint-monitoring.adoc [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+The daemon-set that forwards logs to Anypoint Monitoring expects application containers to write logs to the following locations:
```
Comment (SparshAgarwal) [inline modules/ROOT/pages/use-anypoint-monitoring.adoc:80]: Change content of this section as follows. This is the new section I was talking about earlier

MuleSoft runs Anypoint Monitoring agents alongside mule applications to send telemetry to Anypoint control plane.

**For applications deployed/managed using runtime fabric agent version 2.5.x or lower, two different agents are used as follows:**
- One daemonset per underlying node to transporting logs to Anypoint monitoring control plane
- One sidecar per application replica to transporting metrics to Anypoint monitoring control plane

Following prerequisites are need for optimal daemonset functionality:
1. Application containers should be able to write logs to the following locations:
* `/var/log/pods`
* `/var/log/containers`
* `/var/lib/docker/containers`
2. The daemon-set also creates and writes on the host path `/var/lib/filebeat-data`. 
3. Ensure that
* The Docker daemon config includes the `json-file` logging driver.
* The Anypoint Monitoring log forwarder daemon-set container has write access to the host on the host path `/var/lib/filebeat-data`.

**For applications deployed/managed using runtime fabric versions 2.6.x and above, MuleSoft is unifying telemetry transport to a single sidecar per application replica.** 

- To accommodate added functionality, this sidecar now requires higher resource allocation, ranging from 0.1CPU **(minimum)** to 30% of CPU Limit configured for application replica **(maximum)** to accommodate for needs of the application. Memory consumption may range from 70 MB **(minimum)** to 150 MB **(maximum)**.
- MuleSoft recommends customers allocate appropriate capacity to the underlying cluster calculated using maximum configurations to avoid application deployment failure.

File: modules/ROOT/nav.adoc [hunk 10]
```diff
@@ -26,13 +26,14 @@
...
  ** xref:deploy-maven-4.x.adoc[Deploy Applications Using Maven (Mule 4)]
  ** xref:deploy-maven-3.x.adoc[Deploy Applications Using Maven (Mule 3)]
- ** xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
- ** xref:use-log4j-appender.adoc[Use a Log4j Appender]
  ** xref:protect-app-properties.adoc[Protect App Property Values]
  ** xref:manage-secure-properties.adoc[Protect App Property Values with rtfctl]
  ** xref:app-versioning.adoc[Review Application Versioning]
  ** xref:app-to-app-requests.adoc[Send Application-to-Application Requests]
  ** xref:proxy-deploy-runtime-fabric.adoc[Deploy API Proxies]
+* xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric]
```
Comment (SparshAgarwal) [inline modules/ROOT/nav.adoc:34]: Can we please add a link to this page in release notes to surface this doc change to customers:
"Review the resource allocation changes of anypoint monitoring agent which is used to monitor applications deployed on runtime fabric here<link>"

**Comment by kevintroller** [comment]
Closing as new PR opened https://github.com/mulesoft/docs-runtime-fabric/pull/1242
